rtsp-to-web/Dockerfile

```diff
@@ -3,45 +3,43 @@
 # Example build command from add-on root directory:
 # $ docker build --build-arg BUILD_FROM="homeassistant/amd64-base:latest" --build-arg "BUILD_ARCH=amd64" --build-arg TEMPIO_VERSION=2021.09.0 -t rtsp-to-web rtsp-to-web
```

Comment:
update comment, including removing arguments no longer used e.g. tempio

Reply:
Done. I removed BUILD_FROM and didn't mention BUILD_ARCH as well, because the default may suffice.